code/game/objects/effects/effect_system/smoke.dm

```diff
@@ -319,6 +319,15 @@
 	smoke_can_spread_through = TRUE
 	smoke_traits = SMOKE_XENO|SMOKE_XENO_SANGUINAL|SMOKE_GASP|SMOKE_COUGH
 
+//Weaker version of Sanguinal from defiler for spitter
+/obj/effect/particle_effect/smoke/xeno/lightsanguinal
+	lifetime = 5
+	alpha = 60
```

Comment:
not a sanguinal child when it should be

Reply:
mole said its fine. i dno you and him discuss ples seems like maintainer stuff